config.go

```diff
@@ -75,10 +75,12 @@ type Settings struct {
 	ConfigDir           string                    `yaml:"configdir" bson:"configdir" json:"configdir"`
 	ContainerPools      ContainerPoolsConfig      `yaml:"container_pools" bson:"container_pools" json:"container_pools" id:"container_pools"`
 	Database            DBSettings                `yaml:"database" json:"database" bson:"database"`
+	DevProdOwnedBuckets []string                  `yaml:"dev_prod_owned_buckets" bson:"dev_prod_owned_buckets" json:"dev_prod_owned_buckets"`
```

Comment:
Do we intend to add this field to the API model and the admin page?

Reply:
I added it to the API models and the admin page. Tested it on admin page and it works

Reply:
<img width="382" alt="Screenshot 2024-12-31 at 12 01 17 PM" src="https://github.com/user-attachments/assets/a49b758a-b7ee-4277-a548-0a84909e8aba" />


I added that field via admin page and deleted another one as well

Reply:
I also ran a new task [here](https://spruce-staging.corp.mongodb.com/task/zackary_bisect_s3put_private_s3put_visibility_signed.permissions_private_patch_a337545b280010ef7edb0468d06540e1dea3226f_6774239d0a401e000747531c_24_12_31_17_02_27/files?execution=0), went to the database and changed the keys to be INVALID. This would prove that it's signing with IRSA rather than the keys.

Invalid keys for the artifact:
<img width="688" alt="Screenshot 2024-12-31 at 12 08 06 PM" src="https://github.com/user-attachments/assets/63840181-7ad2-4f5f-9b21-336c723aa1a4" />

Trying to open the file:
<img width="369" alt="Screenshot 2024-12-31 at 12 06 26 PM" src="https://github.com/user-attachments/assets/e9773b80-9674-41c2-b3a8-cced6655b95a" />